src/client/playwright.config.ts

```diff
@@ -5,7 +5,7 @@ const config: PlaywrightTestConfig = {
   testDir: "./e2e",
   /* Maximum time one test can run for. */
   timeout: 50 * 1000,
-  workers: 1,
+  workers: 5,
```

Comment:
I see, I was looking for ways to speed up the tests in case we wanted to test many times, but I guess since the e2e tests run every morning anyway it covers it. I will set it back to 1